api/tests/unit/infrastructure/http/errors-helper_test.js

```diff
@@ -3,6 +3,27 @@ const { expect } = require('../../../test-helper');
 const { getErrorDetails } = require('../../../../lib/infrastructure/http/errors-helper');
 
 describe('getErrorDetails', function () {
```

Comment:
@octo-topi je serais d'avis de renommer ça en serializeHttpErrorResponse, wdyt ?

Reply:
I think it's a better name, indeed

Reply:
done !